pxb-tag/scripts/parse-changelog.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+
+const VERSION = process.argv.slice(2);
+
+console.log(`Version ${VERSION}`);
+console.log();
+
+fs.readFile('CHANGELOG.md', 'utf8', (err, data) => {
+    if (err) {
+        console.log(err);
+        return;
+    }
+
+    const separator = '## v';
+    const releases = data.split(separator);
+    for (const release of releases) {
+        if (release.includes(VERSION)) {
+            fs.writeFile('TAG_CHANGELOG.md', `## v${release}`, (err) => {
+                if (err) {
+                    console.log(err);
+                    return;
```

Comment:
I'm still thinking this should return an exit code here and stop processing if there was a problem writing the file.

Reply:
You're right, exit codes have been added everywhere where an error can occur.